src/sql/plot.py

```diff
@@ -39,10 +38,9 @@ def _summary_stats(con, table, column, with_=None):
     )
     query = template.render(table=table, column=column)
 
-    if with_:
-        query = str(store.render(query, with_=with_))
+    query = sql.connection.Connection.prepare_query(query, con, with_)
```

Comment:
I see this pattern many times:

```
query = sql.connection.Connection.prepare_query(query, con, with_)
values = con.execute(query).fetchone()
```

maybe it'd be better to have the `execute` method call the prepare query logic? this way future contributors won't have to deal with this two step process